documentation/modules/ref-address-space-example-connectors.adoc

```diff
@@ -5,13 +5,17 @@
 [id='ref-address-space-example-connectors-{context}']
 = Address space connector examples
 
-Address space connectors are most commonly used in two ways. One use case is to map addresses on a remote AMQP endpoint into an address space. For example, a broker is running on the host `messaging.example.com` that you want to access by connecting using the {ProductName} endpoints. Another use case is to enable address forwarding. For more information about address forwarding, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-address-example-forwarders-messaging[Address forwarding examples].
+You can federate a `standard` address space type with another AMQP server. Two methods of operation are supported: _remote address connection_ and _message store-and-forward_.
 
-The following address space examples show how you can configure connectors.
+Remote address connection involves mapping addresses on a remote AMQP endpoint into an address space. For example, an AMQP server is running on the host `messaging.example.com` that you want to access by connecting using the {ProductName} endpoints. To enable remote address connection, you need to create an address space connector.
 
-== Connector using SASL PLAIN
+Message store-and-forward involves enabling address forwarding. So, first you need to create an address space connector, and then you need to create an address forwarder for each address. For more information about address forwarding, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-address-example-forwarders-messaging[Address forwarding examples].
```

Comment:
"First you need to create an address space connector. Then, you need to create an address forwarder for each address."